packages/fxa-auth-server/lib/senders/emails/templates/lowRecoveryCodes/index.txt

```diff
@@ -1,8 +1,29 @@
-codes-reminder-title = "Low recovery codes remaining"
+<%- body %>
 
-codes-reminder-description = "We noticed that you are running low on recovery codes. Please consider generating new codes to avoid getting locked out of your account."
+<% if (locals.numberRemaining < 1) { %>
```

Comment:
Sorry @millsoper do you mind updating this conditional too and L10 so it matches the MJML? LGTM otherwise, I see Dan gave the r+ :)

Reply:
done!